# HG changeset patch
# User Rik <rik@octave.org>
# Date 1489508701 25200
#      Tue Mar 14 09:25:01 2017 -0700
# Node ID 9cb69973997dd781cd885a3cf2a907eeb4ad3cba
# Parent  87b6f3606fd44dbe6cefab0c397bf5812fcd857d
Make Ctrl+A default shortcut for "Select All" (bug #50538).

* shortcut-manager.cc (do_init_data): Change default for "Select All"
to QKeySequence::SelectAll.

diff --git a/libgui/src/shortcut-manager.cc b/libgui/src/shortcut-manager.cc
--- a/libgui/src/shortcut-manager.cc
+++ b/libgui/src/shortcut-manager.cc
@@ -110,17 +110,17 @@ shortcut_manager::do_init_data ()
   init (tr ("Load Workspace"), "main_file:load_workspace", QKeySequence ());
   init (tr ("Save Workspace As"), "main_file:save_workspace", QKeySequence ());
   init (tr ("Exit Octave"), "main_file:exit", QKeySequence::Quit);
 
   // edit
   init (tr ("Copy"), "main_edit:copy", QKeySequence::Copy);
   init (tr ("Paste"), "main_edit:paste", QKeySequence::Paste);
   init (tr ("Undo"), "main_edit:undo", QKeySequence::Undo);
-  init (tr ("Select All"), "main_edit:select_all", QKeySequence ());
+  init (tr ("Select All"), "main_edit:select_all", QKeySequence::SelectAll);
   init (tr ("Clear Clipboard"), "main_edit:clear_clipboard", QKeySequence ());
   init (tr ("Find in Files"), "main_edit:find_in_files",
         QKeySequence (ctrl_shift + Qt::Key_F));
   init (tr ("Clear Command Window"), "main_edit:clear_command_window",
         QKeySequence ());
   init (tr ("Clear Command History"), "main_edit:clear_history",
         QKeySequence ());
   init (tr ("Clear Workspace"), "main_edit:clear_workspace", QKeySequence ());
